src/data/landing/becas-partner.es.yml

```diff
@@ -15,6 +15,11 @@ meta_info:
   utm_course: full-stack
   utm_location: madrid-spain
 
+navbar:
+  buttonText: "Solicitar"
+  # url accepts: relative(/apply), absolute(https://google.com) or id(#componentName)
+  url: "#apply_schollarship"
```

Comment:
The url works with links `relative`, `absolute` and `id`(name of the function in landing)